packages/js-moi-logic/src.ts/logic-base.ts

```diff
@@ -217,9 +214,7 @@ export abstract class LogicBase extends ElementDescriptor {
         }
 
         ixObject.send = async (): Promise<InteractionResponse> => {
-            option.fuelLimit = option.fuelLimit != null ? option.fuelLimit : await ixObject.estimateFuel();
-            option.fuelPrice = option.fuelPrice != null ? option.fuelPrice : DEFAULT_FUEL_PRICE;
```

Comment:
why did we remove fuelPrice logic are we handling it in some other place.

Reply:
Resolved In : https://github.com/sarvalabs/js-moi-sdk/pull/75/commits/13b0708ef769be3ddb23de85975b805f491f7163